### PR TITLE
INSTALL-944: Add base linux CI for FindPackage

### PR DESCRIPTION
**Pull request by lulivi**
Fixes #<issue id>

### CODE REVIEW

**Review by jpovedano — DISMISSED** [review]

File: resources/ci/buildConfiguration.groovy [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+ * works of the software solely for use with RTI Connext DDS. Licensee may
+ * redistribute copies of the software provided that all such copies are subject
+ * to this license. The software is provided "as is", with no warranty of any
+ * type, including any warranty for fitness for any purpose. RTI is under no
+ * obligation to maintain or support the software. RTI shall not be liable for
+ * any incidental or consequential damages arising out of the use or inability
+ * to use the software.
+ */
+
+@Library("rticommunity-jenkins-pipelines@feature/INSTALL-944") _
```
Comment (ManuelJNunez) [inline resources/ci/buildConfiguration.groovy:13]: Delete this before merging

File: resources/ci/buildArchitectureConfigurations.groovy [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ * works of the software solely for use with RTI Connext DDS. Licensee may
+ * redistribute copies of the software provided that all such copies are subject
+ * to this license. The software is provided "as is", with no warranty of any
+ * type, including any warranty for fitness for any purpose. RTI is under no
+ * obligation to maintain or support the software. RTI shall not be liable for
+ * any incidental or consequential damages arising out of the use or inability
+ * to use the software.
+ */
+
+@Library("rticommunity-jenkins-pipelines@feature/INSTALL-944") _
```
Comment (ManuelJNunez) [inline resources/ci/buildArchitectureConfigurations.groovy:13]: And this

File: resources/ci/docker/Dockerfile.x64Linux3gcc4.8.2 [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# (c) 2024 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+# RTI grants Licensee a license to use, modify, compile, and create derivative
+# works of the Software.  Licensee has the right to distribute object form only
+# for use with RTI products.  The Software is provided "as is", with no warranty
+# of any type, including any warranty for fitness for any purpose. RTI is under
+# no obligation to maintain or support the Software.  RTI shall not be liable
+# for any incidental or consequential damages arising out of the use or
+# inability to use the software.
+
+from ubuntu:14.04
```
Comment (ManuelJNunez) [inline resources/ci/docker/Dockerfile.x64Linux3gcc4.8.2:10]: Just a question: should you use CentOS 7 for this?

**Review by ManuelJNunez — DISMISSED** [review]

File: resources/ci/docker/Dockerfile.x64Linux3gcc4.8.2 (hunk 3 above)
Comment (lulivi) [reply resources/ci/docker/Dockerfile.x64Linux3gcc4.8.2:10]: this is not in use, we will use our docker containers in the next iteration

**Review by ManuelJNunez — APPROVED** [review]

**Pull request merged**